config/environments/development.rb

```diff
@@ -1,6 +1,11 @@
+require "active_support/core_ext/integer/time"
+
 Rails.application.configure do
-    # Verifies that versions and hashed value of the package contents in the project's package.json
-  config.webpacker.check_yarn_integrity = true
+  # disable integrity check in development
+  config.webpacker.check_yarn_integrity = false
```

Comment:
Question:

Both were in development before upgrade:
  # disable integrity check in development
  config.webpacker.check_yarn_integrity = false

  # Verifies that versions and hashed value of the package contents in the project's package.json
  config.webpacker.check_yarn_integrity = true

Which "DO" we want?

Reply:
the end result would have been false before. so false makes sense

Reply:
@aprilrieger Rez back FALSE